packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/invoke-contract-v2.test.ts

```diff
@@ -0,0 +1,382 @@
+import test, { Test } from "tape";
+import Web3 from "web3";
+import { v4 as uuidV4 } from "uuid";
+import path from "path";
+
+import { LogLevelDesc } from "@hyperledger/cactus-common";
+
+import { PluginKeychainMemory } from "@hyperledger/cactus-plugin-keychain-memory";
+
+import HelloWorldContractJson from "../../../../solidity/hello-world-contract/HelloWorld.json";
+
+import {
+  EthContractInvocationType,
+  PluginLedgerConnectorQuorum,
+  Web3SigningCredentialCactusKeychainRef,
+  Web3SigningCredentialType,
+} from "../../../../../main/typescript/public-api";
+
+import {
+  QuorumTestLedger,
+  IQuorumGenesisOptions,
+  IAccount,
+} from "@hyperledger/cactus-test-tooling";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+
+const logLevel: LogLevelDesc = "INFO";
+const contractName = "HelloWorld";
+const thePath = path.join(
```

Comment:
@AzaharaC Sorry, my example wasn't the best (I need to follow my own advice more). Any chance that you could rename this to something more descriptive like `contractsPath`? That way it would match the parameter name that it is being used for so makes it a little easier on the mind for people who will be reading this code for the first time.

Reply:
Sure! I will change this.